template-only-docs/multiple-applications.md

```diff
@@ -0,0 +1,38 @@
+# Multiple Applications
+
+You can use this template with multiple applications. By default, this template assumes your project has one application named `app`. However, it's straightforward to add additional applications.
```

Comment:
I was actually thinking that we change install-template.sh so that it doesn't add any apps by default. Or we could call install-app at the end of install-template.